lib/tools/android-manifest.js

```diff
@@ -0,0 +1,81 @@
+import { prettyExec } from '../helpers/helpers.js';
+
+let manifestMethods = {};
+
+// android:process= may be defined in AndroidManifest.xml
+// http://developer.android.com/reference/android/R.attr.html#process
+// note that the process name when used with ps must be truncated to the last 15 chars
+// ps -c com.example.android.apis becomes ps -c le.android.apis
+manifestMethods.processFromManifest = async function (localApk) {
+  if(await this.isAaptPresent()) {
```

Comment:
rather than making this a conditional, this could just be an assertion function. it saves us space and from having to use a conditional to throw the error. like:

```js
await this.assertAaaptPresent(); 
```

And that line will throw the error without needing a conditional